docs/api/commands.md

```diff
@@ -1,33 +1,19 @@
+---
+anta_title: ANTA Commands API
+---
 <!--
   ~ Copyright (c) 2023-2025 Arista Networks, Inc.
   ~ Use of this source code is governed by the Apache License 2.0
   ~ that can be found in the LICENSE file.
   -->
 
-# Test definition
-
-![AntaTest UML model](../imgs/uml/anta.models.AntaTest.jpeg)
-
-## ::: anta.models.AntaTest
-
-    options:
-        filters: ["!^_[^_]", "!__init_subclass__", "!update_progress"]
-
-# Command definition
-
-![AntaCommand UML model](../imgs/uml/anta.models.AntaCommand.jpeg)
-
-## ::: anta.models.AntaCommand
-
 !!! warning
     CLI commands are protected to avoid execution of critical commands such as `reload` or `write erase`.
 
       - Reload command: `^reload\s*\w*`
       - Configure mode: `^conf\w*\s*(terminal|session)*`
       - Write: `^wr\w*\s*\w+`
 
-# Template definition
-
-![AntaTemplate UML model](../imgs/uml/anta.models.AntaTemplate.jpeg)
+## ::: anta.models.AntaCommand
```

Comment:
probably worth adding the constant for KNOWN ERRORS somewhere in this doc

Reply:
Yeah, since the guy that wrote this haven't written the doc ;)